Testing/AutomatedUITests/Example/ExampleUITest.py

```
@@ -0,0 +1,558 @@
+    and those implement ``data``/``setData``/``flags`` themselves.
+    """
+    from qtpy.QtCore import QAbstractTableModel, QModelIndex, Qt
+
+    class _ExampleTableModel(QAbstractTableModel):
+        HEADERS = ("Name", "Use", "Note")
+
+        def __init__(self):
+            super(_ExampleTableModel, self).__init__()
+            self.rows = [list(row) for row in _MODEL_ROWS]
+
+        def rowCount(self, parent=QModelIndex()):
```

Ruff (0.16.1)

[warning] Do not perform function call `QModelIndex` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)